### PR TITLE
Bug/default ww

### PR DESCRIPTION
**Pull request by builderjer**
Changed default wakewords to use precise instead of pocketsphinx.  Precise-lite is installed by default on the buildroot image, the manjaro image, and neon.  It is much more "precise" than pocketsphinx

### CODE REVIEW

File: ovos_local_backend/configuration.py [hunk 1]
```diff
@@ -59,18 +59,78 @@
     },
     "default_ww": "hey_mycroft",  # needs to be present below
     "ww_configs": {  # these can be exposed in a web UI for selection
-        "hey_mycroft": {"phonemes": "HH EY . M AY K R AO F T",
-                        "module": "ovos-ww-plugin-pocketsphinx",
-                        "threshold": 1e-90},
-        "hey_jarvis": {"phonemes": "HH EY . JH AA R V AH S .",
```
Comment (JarbasAl) [inline ovos_local_backend/configuration.py:65]: can you please leave the existing pocketsphinx wws? those dont have an equivalent precise model and should remain available

**Comment by JarbasAl** [comment]
I have mixed feelings here, what we really need is a mechanism for devices to report what plugins are installed

this is potentially breaking for vanilla mycroft usage, but i will accept the PR since it already required the ovos pocketsphinx plugin anyway

File: ovos_local_backend/configuration.py (hunk 1 above)
Comment (builderjer) [reply ovos_local_backend/configuration.py:65]: Can we change them to vosk instead? It works WAY better

**Comment by builderjer** [comment]
I agree that it should look for plug-ins, but don't the images come with at least one ww engine that is common to all?

**Comment by builderjer** [comment]
Or I could add precise models too instead of precise-lite? Isn't that what mycroft ships with?

**Comment by JarbasAl** [comment]
> I agree that it should look for plug-ins, but don't the images come with at least one ww engine that is common to all?

this is a standalone project that works with regular mycroft and is not exclusive to our OVOS images, so we cant assume any plugin is installed

mycroft itself lacks a mechanism for reporting installed plugins and removed the ww selection feature from selene with the mk2 release, so i think its fair to consider this a personal backend exclusive feature for now.

In the future I'll look into improving this, for now lets merge this :+1:

File: ovos_local_backend/configuration.py (hunk 1 above)
Comment (JarbasAl) [reply ovos_local_backend/configuration.py:65]: Sure, sounds like a plan! Any other words we want there? Please also remove the engine from the ww name, that should be added in UI directly. The backend manager can add the module itaelf at display time. Im not sure if downstream code will like the parenthesis

**Review by JarbasAl — APPROVED** [review]

**Pull request merged**